### PR TITLE
Feat/apicall get

### PR DESCRIPTION
**Pull request by cadechaney**
## 🛠️ Description of this branch

* Added a call from a function in another file just for the conditionals. This other file I imported, in the conditionals is calling the api.
*  I began the conditionals just practicing some switch conditionals, but we can change them to if statements if that is more readable
* I changed the render in search results, before the map iterator I added a slice so only 6 options are viewed
* I took away some of the options from the dropdown list, we can add back however many we want once we finish the conditional

- [x] New Feature
- [ ] Fix
- [ ] Documentation
- [ ] Tests

## 🧠 Context or Rationale for this code

I took the conditionals to another file in order to keep the main app component as clean and simple as possible. The use of switch conditionals is only for me to practice, I didn't get too far with those so we can change those to if statements if we decide. 

Choose USA and flight song to see current results

## 👀 Checks
- [x] When the project is run locally, the terminal shows no errors or warnings
- [x] Breaks nothing
- [ ] Breaks some things & Creates new issues
- [ ] Code in this branch has been tested  
- [ ] Forgot to write new tests for changes in this code, but it is added to the project board as a ticket. 
- [ ] Code is DRY, reusable, follows SRP and trends toward function purity 
- [ ] I've updated the project board to reflect changes, add issues, or tickets

## ✍🏽 Notes, Questions, What's Next
   The next thing we need to do is either add the possible conditions or make it so we only need one conditional, and interpolate the values. If we merge this we shouldn't have to worry about styling because of the work me and Josh did.

## Screenshot
Screen Shot 2023-05-31 at 12 49 24 AM

### CODE REVIEW

File: src/FetchConditonal/FetchConditional.tsx
Comment (sdmisra) [inline src/FetchConditonal/FetchConditional.tsx:1]: This is really neat - I think we'd call this a 'cleaner' function that is cleaning up our input values to be more useful for the API call. 

Switch statements are pretty new to me, so I'd really enjoy if we walked through these changes together to make sure we talk about what is happening (later today or whenever). 

I am curious about lines 20 and 29 - I recognize what the data.reordings?.filter. is doing (filtering as long as there are recordings coming back in the data) but i dont quite know what the double '?' symbol means! (The '?? []') at the end.

Awesome work though Cade, thanks for burning the late night oil on all of this!

**Review by sdmisra — APPROVED** [review]
Great stuff! MVP is getting closer and closer.

**Review by CBradsen — APPROVED** [review]
Great work digging into all this new stuff and the API Cade! 

This typescript bit: Promise<Recording[]> is interesting! 
I didn't think about how TS wants even the promise to be defined. 

I look forward to learning more about how the switch / case  and break/ default work here.

**Review by jmartin777 — COMMENTED** [review]
After talking in standup today the  code is clear, concise, and follows new programming practices. It effectively fetches and filters recordings based on the given parameters after viewing it this morning.
TypeScript is used to define types for the API response (ApiResponse) and the recording object (Recording). This is great for catching potential type errors .
Really great refactoring here Cade and progressively seeing our results more accurately.

FetchConditional function seems to have two identical cases in the switch statement. This could potentially lead to unexpected behavior or incorrect results.

**Pull request merged**